dev/com.ibm.ws.microprofile.config/resources/com/ibm/ws/microprofile/config/resources/Config.nlsprops

```diff
@@ -94,6 +94,28 @@ no.such.element.CWMCG0015E=CWMCG0015E: The property {0} was not found in the con
 no.such.element.CWMCG0015E.explanation=The requested property did not exist in any of the configured sources.
 no.such.element.CWMCG0015E.useraction=Add the property to one of the existing sources or add a new source that contains the property.
 
+# Unable to cancel asynchronous update thread.
+future.update.not.cancelled.CWMCG0016E=CWMCG0016E: The server is unable to cancel the asynchronous update thread for {0}.
```

Comment:
The server is unable to cancel the asynchronous update thread for the {0} type.
Question: Type of what? Specify the object type.